apps/browser/src/browser/browserApi.ts

```diff
@@ -217,7 +217,7 @@ export class BrowserApi {
   static reloadOpenWindows() {
     const views = chrome.extension.getViews() as Window[];
     views
-      .filter((w) => w.location.href != null)
+      .filter((w) => w.location.href != null && !w.location.href.includes("background.html"))
```

Comment:
```suggestion
      .filter((w) => w.location.href !== null && !w.location.href.includes("background.html"))
```

Still going through our code style guidelines. Do we have a preference regarding coercion? In the past, I've preferred to do a strict equality check... but I do know some teams don't approach boolean checks in that manner. (Kyle Simpson, for instance, is a huge fan of allowing for type conversions using coercion).

Reply:
We use strict checks _except_ for null, since we want to match both `undefined` and `null`, in general